### PR TITLE
Start using `ndarray`

### PR DESCRIPTION
**Pull request by lnicola**
Includes #4. We start using `ndarray` here, but not in the most interesting place.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -106,8 +103,11 @@ pub fn thinning_zs(im: &mut [u8], w: usize, h: usize) {
     loop {
         dbg!(iter);
         let mut diff = false;
-        if dbg!(thinning_zs_iteration(im, 0, 0, w, h, w, h)) {
-            diff = true;
+        for r in 1..5 {
+            if dbg!(thinning_zs_iteration(im, 0, 0, w, h, w, h, r)) {
+                thinning_zs_post(im, 0, 0, w, h, w);
```
Comment (latot) [inline src/main.rs]: Doing thinning_zs_post here, is not the same as GRASS, at least there is executed every time a column is ready. this does not needs to be bad at all, just as a note.

Comment (lnicola) [reply src/main.rs]: Yeah, GRASS does this after each row, but it keeps a copy of the previous one, so it doesn't mix old and new values inside one of the 4 iterations.

File: src/main.rs [hunk 2]
```diff
@@ -250,6 +272,7 @@ fn main() -> Result<(), Box<dyn Error>> {
     // let height = 599280;
 
     // thinning_zs(im, width, height);
+    let (tile_width, tile_height) = (8, 8);
```
Comment (latot) [inline src/main.rs]: is this right?

Comment (lnicola) [reply src/main.rs]: No, have you seen https://github.com/latot/thinning/pull/4#pullrequestreview-1386547199?

Comment (latot) [reply src/main.rs]: yes, was just... is right this to be here?

Comment (lnicola) [reply src/main.rs]: I added it as a test, since it wasn't broken before. Just removed it now.

Comment (latot) [reply src/main.rs]: I think, we could add some tests, including this as a param

Comment (lnicola) [reply src/main.rs]: In some other PR :-).

**Pull request merged**